app_legacy/Helpers/render/game.php

```diff
@@ -190,6 +190,45 @@ function renderGameCard(int|array $game): string
     return $tooltip;
 }
 
+function RenderGameSort(bool $isFullyFeaturedGame, ?int $flags, int $officialFlag, int $gameID, ?int $sortBy): void
+{
+    echo "<div class='py-3'><span>";
+    echo "Sort: ";
+
+    $flagParam = ($flags != $officialFlag) ? "f=$flags" : '';
+
+    $sortType = ($sortBy < 10) ? "^" : "<sup>v</sup>";
+
+    $sort1 = ($sortBy == 1) ? 11 : 1;
+    $sort2 = ($sortBy == 2) ? 12 : 2;
+    $sort3 = ($sortBy == 3) ? 13 : 3;
+    $sort4 = ($sortBy == 4) ? 14 : 4;
+    $sort5 = ($sortBy == 5) ? 15 : 5;
+
+    $mark1 = ($sortBy % 10 == 1) ? "&nbsp;$sortType" : "";
+    $mark2 = ($sortBy % 10 == 2) ? "&nbsp;$sortType" : "";
+    $mark3 = ($sortBy % 10 == 3) ? "&nbsp;$sortType" : "";
+    $mark4 = ($sortBy % 10 == 4) ? "&nbsp;$sortType" : "";
+    $mark5 = ($sortBy % 10 == 5) ? "&nbsp;$sortType" : "";
+
+    if ($isFullyFeaturedGame) {
+        echo "<a href='/game/$gameID?$flagParam&s=$sort1'>Normal$mark1</a> - ";
+        echo "<a href='/game/$gameID?$flagParam&s=$sort2'>Won By$mark2</a> - ";
+        // TODO sorting by "date won" isn't implemented yet.
+        // if(isset($user)) {
+        //    echo "<a href='/game/$gameID?$flagParam&s=$sort3'>Date Won$mark3</a> - ";
+        // }
+        echo "<a href='/game/$gameID?$flagParam&s=$sort4'>Points$mark4</a> - ";
+        echo "<a href='/game/$gameID?$flagParam&s=$sort5'>Title$mark5</a>";
+    }
+    else {
+        echo "<a href='/game/$gameID?$flagParam&s=$sort1'>Achievements, Title$mark1</a> - ";
```

Comment:
No need to mention the secondary sort - that will just confuse users.

Though I would recommend changing is to "Has Achievements" or something similar as the implication is that it's sorted by number of achievements.

Reply:
What do you think of "Default" instead of "Normal" and "Has Achievements"? I have been struggling for something that looks right

Reply:
"Normal" would at least be consistent with the game page.

Reply:
For full disclosure, I had previously given feedback that "Normal" was ambiguous and confusing wording for this sort. If I were to give an opinion here, I think "Default" probably works better.